src/Core/src/Handlers/ScrollView/ScrollViewHandler.Android.cs

```diff
@@ -32,14 +32,18 @@ protected override void DisconnectHandler(MauiScrollView nativeView)
 			nativeView.ScrollChange -= ScrollChange;
 		}
 
+		Size _previousDesiredSize;
+
 		public override Size GetDesiredSize(double widthConstraint, double heightConstraint)
 		{
 			var result = base.GetDesiredSize(widthConstraint, heightConstraint);
 
-			if (FindInsetPanel(this) == null)
+			if (result != _previousDesiredSize && FindInsetPanel(this) == null)
```

Comment:
If the measure/layout pass has been triggered by something in the ScrollView content, not measuring it here could mean that it doesn't get updated/redrawn properly. I think we should avoid adding this caching here.

Reply:
Ok, this have a small impact, but let me review alternatives. If not, we can remove this, merge and continue improving in another PR.

Reply:
@hartez Could you review it again?